pkg/provision/storage/asyncStorage.go

```diff
@@ -249,3 +256,13 @@ func (*AsyncStorageProvisioner) getAsyncWorkspaceCount(api provision.ClusterAPI)
 	}
 	return started, total, nil
 }
+
+func checkConfigured() error {
+	if images.GetAsyncStorageServerImage() == "" {
+		return fmt.Errorf("asynchronous storage server image is not configured")
```

Comment:
user does not have the ability to fix that issue. Maybe it makes sense to add some reference to admin, like `Contact admin` ?

Reply:
Added a note about contacting an admin higher up to avoid a Go warning about punctuation in a string :upside_down_face: